RusimbiP/Teamwork
# #168821962 Configure server and development environment

#### What does this PR do?
Have the server and development environment configured
#### Description of Task to be completed?
Have the following configured:
- Eslint
- Babel
- Travis
- Coveralls
- Server
- Tests(Mocha and chai)

#### How should this be manually tested?
By cloning this repo cd into **Teamwork** and run **npm i**

- run **npm start** to see if the server can start
- check the tests by running **npm test**
- check the configuration of eslint in **.eslintrc.json** file

#### Any background context you want to provide?
N/A
#### What are the relevant pivotal tracker stories?
#168821962 
#### Screenshots (if appropriate)
N/A
#### Questions:
N/A
